fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1650,6 +1670,145 @@ opt_user_role ::=
     :}
     ;
 
+opt_password_option ::=
+    opt_passwd_history_policy:historyPolicy
+    opt_passwd_expire_policy:expirePolicy
+    opt_passwd_reuse_policy:reusePolicy
+    opt_failed_login_attempts:loginAttempts
+    opt_password_lock_time:passwdLockTime
+    opt_lock_account:lockAccount
+    {:
+        RESULT = new PasswordOptions(expirePolicy, historyPolicy, reusePolicy, loginAttempts, passwdLockTime, lockAccount);
+    :}
+    ;
+
+opt_passwd_expire_policy ::=
+    {:
+        RESULT = -2L;
```

Comment:
Does `cup` has constant variable to identify `-2`? Or import a static value from embedded java code.

Reply:
Done, use PasswordOptions.UNSET